backend/routes/loginAuth.ts

```diff
@@ -1,11 +1,10 @@
 import express from "express";
 const loginAuthRouter = express.Router()
-import { Strategy as LocalStrategy } from 'passport-local';
 import passport from 'passport'
-import prisma from '../utils/prismaClient'
-import crypto from 'crypto';
 
-const db = prisma;
+loginAuthRouter.get('/', (req, res) => {
+    res.status(400).send('failed login')
+})
```

Comment:
Whats this for?

Reply:
Oh this was just to test local strategy `failureRedirect: '/login'`, I will remove it.